app/src/main/java/com/pwhs/quickmem/presentation/app/notification/NotificationUiAction.kt

```
@@ -0,0 +1,4 @@
+package com.pwhs.quickmem.presentation.app.notification
+
+sealed class NotificationUiAction {
+}
```

detekt

[warning] The class or object NotificationUiAction is empty.

(detekt.empty-blocks.EmptyClassBlock)